rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -356,4 +371,9 @@ private void loadInitialBootNodes(List<String> nodes) {
 
         return ret;
     }
+
+    @VisibleForTesting
+    public NodeChallengeManager getChallengeManager() {
```

Comment:
Injecting this would be cleaner. You should tackle it if it doesn't interfere with your other tasks.

Reply:
I will address this on another PR